### PR TITLE
Add ubuntu cis profile

### PR DESCRIPTION
**Pull request by willumpie**
#### Description:

Add a CIS profile for Ubuntu 1804.

#### Rationale:
First start adding the profile and the move forward by enabling rules and creating new rules.

### CODE REVIEW

**Comment by jan-cerny** [comment]
@willumpie Can you rebase and remove the extra commits from this PR, please?

**Comment by willumpie** [comment]
Rebased on Master.

This profile is copied from RedHat's CIS profile and manually checked against the Ubuntu1804 CIS profile (CIS Ubuntu Linux 18.04 LTS Benchmark v1.0.0 - 08-13-2018)

I'd like to start enabling rules for Ubuntu 1804 in other branches and small commits and evaluate the profile on a Ubuntu 1804 system.

**Comment by willumpie** [comment]
Sorry, Missed one commit...

**Comment by jan-cerny** [comment]
@openscap-ci ok to test

**Comment by jan-cerny** [comment]
@openshift-ci-robot 

/ok-to-test

**Comment by mildas** [comment]
**Changes identified:**
Profile cis on ubuntu1804:
 Newly added profile.

**Recommended tests to execute:**
 build_product ubuntu1804
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ubuntu1804-ds.xml cis

**Comment by mildas** [comment]
**Changes identified:**
Profile cis on ubuntu1804:
 Newly added profile.

**Recommended tests to execute:**
 build_product ubuntu1804
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ubuntu1804-ds.xml cis

**Comment by mildas** [comment]
**Changes identified:**
Profile cis on ubuntu1804:
 Newly added profile.

**Recommended tests to execute:**
 build_product ubuntu1804
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ubuntu1804-ds.xml cis

**Comment by mildas** [comment]
**Changes identified:**
Profile cis on ubuntu1804:
 Newly added profile.

**Recommended tests to execute:**
 build_product ubuntu1804
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ubuntu1804-ds.xml cis

File: ubuntu1804/profiles/cis.profile [hunk 1]
```diff
@@ -0,0 +1,910 @@
+documentation_complete: true
+
+title: 'CIS Ubuntu 19.08 LTS Benchmark'
```
[the remark this answers is not in the document]
Comment (willumpie) [reply ubuntu1804/profiles/cis.profile]: Of course. That is weird. I have changed it.

**Comment by redhatrises** [comment]
LGTM

**Pull request merged**